tests/sqllogictests/suites/query/02_function/02_0048_function_semi_structureds_parse_json

```diff
@@ -222,3 +222,28 @@ select to_variant(a), to_variant(b), to_variant(c) from t3
 statement ok
 DROP DATABASE db1
 
+
+statement ok
+DROP DATABASE IF EXISTS db2
+
+statement ok
+CREATE DATABASE db2
+
+statement ok
+USE db2
+
+statement ok
+CREATE TABLE IF NOT EXISTS test(a bitmap) Engine = Fuse
+
+statement ok
+insert into test values('100,200'),('50,60,90');
```

Comment:
move those tests to `tests/sqllogictests/suites/query/02_function/02_0002_function_cast` is better.

Reply:
Done.